### DISCUSSION TITLE
Additional quests for lean-to

### DISCUSSION THREAD

**Discussion by ERYpTION** [streetcomplete/StreetComplete#5611] [Ideas]
Would it be possible to add a few additional quests for ’amenity=shelter + shelter_type=lean_to’?

I would suggest:

fee=* (Do you need to pay to use this?)
access=* (Who is allowed to use this? It may be free or paid)
capacity=* (number of adult persons able to sleep in the lean-to)
reservation=* (Do you need to book this in advance?)

Thanks.

**Comment by matkoniecz** [comment]
> fee=* (Do you need to pay to use this?)

I expect vast, vast majority to be free. Where you get paid ones?

**Reply by ERYpTION** [reply]
In Denmark, it's not uncommon for the privately owned (but publicly available) shelters to ask for a small fee. I have seen this too in Sweden.

**Reply by HolgerJeromin** [reply]
https://wiki.openstreetmap.org/wiki/Tag:shelter_type%3Dlean_to
says:
Required properties :
* Free rent, open to the public

**Reply by mnalis** [reply]
`fee=*` tag for `lean_to` is also documented in the wiki, and does seem to be tagged sometimes (mostly in Scandinavia and around USA east coast)...  However, usage seem quite low. Out of [16k+ shelter_type=lean_to](https://taginfo.openstreetmap.org/tags/shelter_type%3Dlean_to#overview), only [64 have fee=yes](https://overpass-turbo.eu/s/1L4d) worldwide. That is less than `0.4%`. So definitely would be too spammy worldwide, as it would fail StreetComplete [Quest Guidelines](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md):

_"💤 No spam: It must be possible to determine whether the quest should reasonably be asked. A quest which is to be answered in 99% of the time with the same answer is not a good quest, don't bore the users."_

For Denmark itself (as one of the heavy users), it occurs order of magnitude more often ([37](https://overpass-turbo.eu/s/1L4e) out of [1198](https://overpass-turbo.eu/s/1L4f) have `fee=yes`) but that is still relatively low `~3.1%`. IMHO, ideally the quest would have at least about 10% of the non-default answer to be useful (but that is not a hard number).

---

So, I'm not sure it would make a very good quest. Possible alternatives:

- when you find such shelter with fee, long-press in StreetComplete and leave an OSM Note that fee is required (so you, or someone else, can later add it to the main OSM database). If there are not that many, that is probably the easiest option, but has disadvantage that it takes time for _post-processing_ those Notes...
- one try might suggesting such quests that fail StreetComplete guidelines to [SCEE](https://wiki.openstreetmap.org/wiki/SCEE) (StreetComplete _"Expert Edition"_) fork if they're interested in using it, as it has more flexible quest inclusion guidelines (especially for disabled-by-default quest, and especially if someone is willing to write the quest, which should be [easy](https://github.com/streetcomplete/StreetComplete/blob/master/CONTRIBUTING_A_NEW_QUEST.md) for what is asked here - mostly copy/paste and changing texts; no programming knowledge required really. And in both projects there are people willing to help newcomers learn the ropes!)
- one might use some other editor to add that information. I'd probably use [SCEE](https://wiki.openstreetmap.org/wiki/SCEE) `Show/edit tags` functionality (in its expert mode) as I mostly use that variant of SC for mapping, or [OsmAnd](https://wiki.openstreetmap.org/wiki/OsmAnd) `Modify POI` functionality (as I use OsmAnd for navigation and track recording all the time too). [Every Door](https://wiki.openstreetmap.org/wiki/Every_Door) is also popular mobile editor for such POIs which would be "too spammy" for Quest-based approach.

**Reply by ERYpTION** [reply]
Nonetheless some privately owned lean-tos that are open to the public do ask for a fee. How should those be tagged then? 
On the wikipage it also says under 'Useful combination': fee=* and charge=*

Anyway, I only made this as a suggestion based on what I see in my area. Perhaps because they are becoming "too" popular here, even for the lean-tos on public land it is getting more and more common for them to be bookable (but free) while private ones ask for a small fee.

**Reply by ERYpTION** [reply]
@mnalis Makes sense not to include the quests if the fee-situation in Denmark is not normal. I will find some other way to add the tags, when I come across such shelters. 
I read somewhere that 2/3 of the shelters on public land in Denmark are bookable but come to think of it then perhaps it also doesn't make sense to do a reservation-quest for this since it will almost be the norm.

**Reply by mnalis** [reply]
- ref. https://github.com/openstreetmap/id-tagging-schema/issues/1350

**Reply by ERYpTION** [reply]
Would it then also be possible to do a fee- and reservation quest in SC that is only active in Denmark (and perhaps Sweden)? From what I can tell currently 10% of the Danish lean-tos are tagged with fee=yes and 25% with reservation=*.

**Reply by mnalis** [reply]
@ERYpTION  It _might_ be. Technically it is possible, e.g. there is already a _"Can you pay by cash"_ quest which is only available in Sweden. Main issue is likely to be _"[🕓 Effort vs impact](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md)"_, though, but effect of that rule is reduced if someone is willing to write the quest. 

You should choose one of those tags with higher usage (e.g. [reservation](https://overpass-turbo.eu/s/1S0s) which seems Denmark-only) to start with and [suggest a new quest](https://github.com/streetcomplete/StreetComplete/issues/new?assignees=&labels=&projects=&template=quest-suggestion.md), filling the template with as much information as you have (and referencing this discussion). If it gets accepted and written, then you can suggest other one (no point wasting effort for second one if the first one gets rejected).

**Reply by paulklie** [reply]
Note that fee was already rejected for shelter_type = basic_hut, which is more likely to have a fee: https://github.com/streetcomplete/StreetComplete/issues/6358#issuecomment-3004209996

**Comment by paulklie** [comment]
> capacity=* (number of adult persons able to sleep in the lean-to)

I have created a PR for shelter capacity: https://github.com/streetcomplete/StreetComplete/pull/6442
However I have left out lean-to since in my expierience it is not always clear
